modules/contacts/client/views/directives/tr-contact.client.view.html

```diff
@@ -12,6 +12,11 @@ <h4>
     <span ng-if="::contactCtrl.contact.confirmed === false">Requested</span>
     <time ng-bind="::contactCtrl.contact.created | date:'mediumDate'"></time>
   </small>
+  <div>
+    <small class="text-muted">
```

Comment:
The `small` tag could be replaced with css? However the `small` tag is used in multiple places, therefore it perhaps doesn't matter?

Reply:
yeah it doesn't matter for now, but maybe we can replace all `<small>` tags at some point

Reply:
Could all 3 tags be replaced with just one?

`<small ng-bind="..."></small>`